doc/manuals/orion-api.md

```diff
@@ -1484,6 +1485,29 @@ From original NGSIv2 specification section "Builtin metadata", regarding `action
 Current Orion implementation supports `update` and `append`. The `delete` case will be
 supported upon completion of [this issue](https://github.com/telefonicaid/fiware-orion/issues/1494).
 
+## Ambiguous subscription status `failed` not used
```

Comment:
A future task would need to review subscription payloads examples that include `status` with `failed` value

Reply:
Noted in https://github.com/telefonicaid/fiware-orion/pull/4141